### PR TITLE
XW-4735 | Debug resource for logger

### PR DESCRIPTION
**Pull request by bkoval**
@Wikia/x-wing

### CODE REVIEW

**Review by hakubo — APPROVED** [review]

File: app/utils/analyzeTrackedUrl.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default function analyzeTrackedUrl(params) {
 	const actualUrl = window.location.href;
 
 	if (actualUrl.indexOf(gaPage) === -1) {
-		logEvent('GA url does not match window.location', Object.assign({
+		logDebug('GA url does not match window.location', Object.assign({
```
Comment (latata) [inline app/utils/analyzeTrackedUrl.js:18]: Do we still need this logging? if yes let's create a ticket to remove it in the future.

File: app/routes/wiki-page.js [hunk 2]
```diff
@@ -148,7 +148,7 @@ export default Route.extend(
 				}
 			} else {
 				this.get('logger').warn('Unsupported page');
-				logEvent('Model is missing (unsupported page)');
+				logError('Model is missing (unsupported page)');
```
Comment (latata) [inline app/routes/wiki-page.js]: Do we still need this logging? if yes let's create a ticket to remove it in the future.

File: app/routes/wiki-page-error.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ export default Route.extend(
 		 */
 		renderTemplate(controller, error) {
 			if (!this.get('fastboot.isFastBoot')) {
-				logEvent('Wiki-page route error substate', error);
+				logError('Wiki-page route error substate', error);
```
Comment (latata) [inline app/routes/wiki-page-error.js:15]: Do we still need this logging? if yes let's create a ticket to remove it in the future.

File: app/routes/wiki-page.js (hunk 2 above)
Comment (bkoval) [reply app/routes/wiki-page.js]: Seems like there are no events of this kind in the event-logger index in Kibana. I'll remove it.

File: app/utils/analyzeTrackedUrl.js (hunk 1 above)
Comment (bkoval) [reply app/utils/analyzeTrackedUrl.js:18]: Yeah, we need to preserve it to see if our fixes around tracked GA urls are working. https://wikia-inc.atlassian.net/browse/XW-4850

File: app/routes/wiki-page-error.js (hunk 3 above)
Comment (bkoval) [reply app/routes/wiki-page-error.js:15]: I'd leave it in place - it gives us an overview of how many and in which places people are landing in error substates. Might become useful.

**Review by latata — APPROVED** [review]

**Pull request merged**